Unbox.swift

```diff
@@ -649,26 +710,26 @@ private extension UnboxableWithContext {
 }
 
 private extension Unboxer {
-    static func unboxerFromData(data: NSData, context: Any?) throws -> Unboxer {
+    static func unboxerFromData(data: NSData, context: Any?, skipInvalidData: Bool = false) throws -> Unboxer {
```

Comment:
I see, that makes sense and seems more conscious. I just didn't want to impact the rest of the code.